tcms/testplans/views.py

```diff
@@ -759,7 +780,8 @@ def post(self, request, plan_id):
 class DeleteCasesView(View):
     """Delete selected cases from plan"""
 
-    def post(self, request, plan_id):
+    @staticmethod
+    def post(request, plan_id):
```

Comment:
Class based view have the signature as shown above, however they may not use the self variable. You should revert the changes and open an issue against pylint-django with this code snippet so we can teach pylint-django to ignore these error messages for class based views.